backend/main.py

```
@@ -1,45 +1,98 @@
 
-app.include_router(router, prefix="/api")  
+
+# Global application instance
+app = DevRAIApplication()
+
+async def main():
+    """Main entry point"""
+
+    # Setup signal handlers for graceful shutdown
+    def signal_handler(signum, frame):
+        logger.info(f"Received signal {signum}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)